pyclaragenomics/README.md

```diff
@@ -20,8 +20,8 @@ pip install pyclaragenomics-cuda10-1
 ```
 
 Details of the packages are available here -
-- https://pypi.org/project/pyclaragenomics-cuda10-0/#description
-- https://pypi.org/project/pyclaragenomics-cuda10-1/#description
+- https://pypi.org/project/pyclaragenomics-cuda-10-0
```

Comment:
This is not mine - I'm going to leave it.

Reply:
@tijyojwad @vellamike @ohadmo Anyone know where does this change comes from?

Reply:
This change was merged into `master` branch and not `dev-v0.5.0`. Maybe this branch merged in master at some point?

Reply:
Oops, that probably was me then - I'll revert this.

Reply:
Reverted in 0562f1c.

Reply:
Thanks @tijyojwad! Any reason why this changed wasn't merged in `dev-v0.5.0` as well? I guess the idea was that `dev-v0.5.0` will get merged into `master` soon and `dev-v0.6.0` will be branched off `master`

Reply:
@mimaric it was absolutely supposed to be merged into master but slipped my mind. Needed manual merging because of the version update in master (which overwrites version in dev-v0.5.0). And since we merge master into dev before releasing, I figured I'd just do it before the release :)